packages/angular-ui-components/package.json

```diff
@@ -25,49 +25,55 @@
 	"scripts": {
 		"clean": "rm -rf node_modules dist",
 		"start": "ng serve",
-		"build": "ng build --prod",
+		"build": "ng build --configuration production",
 		"build:watch": "ng build --watch",
```

Comment:
isn't necessary to add `--configuration development` here?

Reply:
We don't have a development configuration, and I don't think there is a difference in `tsconfig.lib.prod.json`. Or should something be different?

Reply:
Forget. We don't need this. But by default the angular.json will use `tsconfig.json`. Only when configuration flag is set to `production` will gonna use `tsconfig.lib.prod.json`